components/SkillSelect.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Text, TouchableOpacity, View, TextInput } from 'react-native';
+
+import styles from '../constants/SkillSelect';
+
+const Skills = {
```

Comment:
Let's move this somewhere more discoverable, just in case we need it somewhere else. Maybe under `constants/`?